features/storage/csi_clone.feature

```diff
@@ -0,0 +1,45 @@
+Feature: CSI clone testing related feature
+  """
+  This test can only work on a Cluster which has CSI driver deployed, OSP has csi deploy by default from 4.7. 
+  From 4.7, OSP CSI driver was installed by default. Its storage class name is standard-csi.
+  Use "oc get sc" can find all deployed stroage class.
+  """
+  
+  # @author jianl@redhat.com
+  # @case_id OCP-27615
+  @admin
```

Comment:
Could you point where is the admin privilege needed ?

Reply:
Good question, actually I removed it some days ago, but it failed, and at that time I could not figure out what caused that failure.  But right now we know the failure is caused by storage's strategy. I will remove @admin tag if it does not block the test. Till now i have no cluster ready, i will verify it when my cluster setup. @jhou1

Reply:
Sure, since removing the tag doesn't touch any test scenario, feel free to skip the test to save your time.
To explain it in more detail, `@admin` enables the user to create/update resources outside of the current namespace. I looked at the test that the operations of creating pods/pvc are within the user's namespace, so `@admin` is not necessary, thanks Liang for pointing that out.

Reply:
Yes, because at very beginning, I try to deploy csi with existing code, it requires admin permission, so the admin tag left here.